scripts/submit_csv.sh

```diff
@@ -44,7 +44,7 @@ ARGUMENTS="EVGEN/\$(file) \$(ext) \$(nevents) \$(ichunk)"
 source $(dirname $0)/set_bg_mix_var.sh
 
 # construct environment file
-ENVIRONMENT=environment-$(date --iso-8601=minutes).sh
+ENVIRONMENT=environment-$(date --iso-8601=ns).sh
```

Comment:
I'd have gone with `--iso-8601=seconds` but I see you're planning for productivity!